src/dashboard/src/templates/archival_storage/_archival_storage_search_form.html

```diff
@@ -5,9 +5,7 @@
     <div id="search_form_container" class="clearfix"></div>
     <div id="search_form_submit_container" class="clearfix">
       <span style="margin-left: 10px">{% trans "Show files?" %}</span>
-      <span style="margin-left: 5px"><input id="id_show_files" type="checkbox" {{ checked_if_in_file_mode }} /></span>
-      <span style="margin-left: 10px">{% trans "Show AICs?" %}</span>
```

Comment:
Good catch! I meant to include a note about this in the PR description and forgot.

Contrary to what I assumed would happen, the "Show AICs" checkbox doesn't include or exclude AICs from the search results (they are always included), but instead determines whether the "AIC" column in the search results table. Given that column visibility is now the domain of the DataTable and new visibility selector, I removed this checkbox intentionally so that we don't have two places potentially contradictory UI elements controlling the visibility of the same column.

I'll run this by the product owner and other internal stakeholders on Slack for validation, as you're right, this should just be a surprise for folks.

Reply:
That makes sense to me!! "Show AICs" has always been unnecessarily confusing anyway :confounded:

Reply:
Sounds good. It's way more intuitive now too. I think that rationale and impact should just be placed in the PR comment, and likely testing notes associated with the change (i,e, users can expect a slight difference in screens!). If screenshots in the docs are impacted by the change, which it seems like they will be anyway with the extent of the data tables improvements, then the removal of the check-box is a good detail to highlight to make sure that it's included. We actually have a `docs needed` label which we can use for the issue associated with this one.

Reply:
Will do, thanks!